From c4464e8425c6211430a837df8fcbd7410fa48041 Mon Sep 17 00:00:00 2001
From: ahmed_hussien <a.hussien.91@gmail.com>
Date: Sat, 18 Mar 2023 05:36:39 +0000
Subject: [PATCH] add print

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 372c70e519ce..d7bb638d5b7d 100644
--- a/init/main.c
+++ b/init/main.c
@@ -779,6 +779,7 @@ asmlinkage __visible void __init start_kernel(void)
 	acpi_subsystem_init();
 	arch_post_acpi_subsys_init();
 	sfi_init_late();
+	printk("Hellooooooooooooooooo from kernelllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll");
 
 	/* Do the rest non-__init'ed, we're now alive */
 	arch_call_rest_init();
-- 
2.34.1

